tests/integration/test_ethereum_tester.py

```diff
@@ -252,7 +252,7 @@ class TestEthereumTesterEthModule(EthModuleTest):
         EthModuleTest.test_eth_sign_transaction_ens_names, ValueError
     )
     test_eth_submitHashrate = not_implemented(EthModuleTest.test_eth_submitHashrate, ValueError)
-    test_eth_submitWork = not_implemented(EthModuleTest.test_eth_submitWork, ValueError)
+    test_eth_submit_work = not_implemented(EthModuleTest.test_eth_submit_work, ValueError)
 
```

Comment:
I think to get your ethtester tests passing, you'll need to add `test_eth_submitWork_deprecated` here.

Reply:
okay. Thanks for catching that. Added that change